static/src/js/actions/urlQuery.js

```diff
@@ -73,13 +73,15 @@ export const changePath = (path = '') => async (dispatch, getState) => {
 
   const [pathname, queryString] = path.split('?')
 
+  let decodedParams
+
   // If query string is a projectId, call getProject
   if (queryString && queryString.indexOf('projectId=') === 0) {
     const requestObject = new ProjectRequest(undefined, earthdataEnvironment)
 
     const { projectId } = parse(queryString)
 
-    const projectResponse = requestObject.fetch(projectId)
+    await requestObject.fetch(projectId)
```

Comment:
I don't like the combination of async/await and then/catch. This `await` should also prompt a change to using a try/catch where anything in the `then` block` is executed after the `await` within a `try` and the `catch` is moved to the `catch` block of the `try`.